### ISSUE TITLE
Plugins download is broken in 1.0

### ISSUE THREAD

**Issue by ileasile** [JetBrains/gradle-intellij-plugin#688]
After upgrading from 0.7.3 to 1.0, plugin download from custom repositories fails for me. Judging by debug log, pom is resolved successfully, but zip file cannot be resolved because artifact URL is ill-formed (its absolute path is added to the repo URL):
```
2021-06-01T19:40:15.669+0300 [INFO] [org.gradle.internal.resource.transport.http.HttpClientHelper] Resource missing. [HTTP HEAD: https://buildserver.labs.intellij.net/guestAuth/app/rest/builds/buildType:(id:ijplatform_master_PyCharm_InstallersBuild),number:212.3724,branch:default:any/artifacts/content/PY-plugins/https:/buildserver.labs.intellij.net/guestAuth/app/rest/builds/buildType:(id:ijplatform_master_PyCharm_InstallersBuild),number:212.3724,branch:default:any/artifacts/content/PY-plugins/auto-uploading/python-212.3724.zip
```

**Comment by ileasile** [comment]
It seems that the problem is here: https://github.com/JetBrains/gradle-intellij-plugin/commit/c9cfd42cbd7276a966213c2fd23b1ec33caf97f1#r51585587

**Comment by hsz** [comment]
Thanks, Ilya – that was a neat investigation!
The issue is fixed and will be released soon with v1.0.1.
